updated branch no fork

Purple906 opened the pull request:
branch of mobile site for BB

Purple906 committed:
updated branch no fork

branch of mobile site for BB

mysterycommand commented on index.html:
```diff
@@ -0,0 +1,116 @@
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <title>Becky Bosshard</title>
```
Bosshar**t**.

mysterycommand commented on index.html:
```diff
@@ -0,0 +1,116 @@
+        <a href="#">Phone</a>
+    </nav>
+    <h1>
+        Becky Bosshard
```
Bosshar**t**.

mysterycommand commented:
Okay, some somewhat tedious changes:
- [x] move `index.html` to `source/index.html`
- [x] move/rename `stylesheet.css` to `source/main.css`
- [x] move all the `BB_photo*.png` to `source/images/bb-photo-*.png` (might rename these to be a little more descriptive at some point, but lower case and with dashes is good enough for now)
- [x] move/rename `Facebook.png` to `source/icons/facebook.png` (lowercase, we'll keep other social icons here as well)
- [x] be sure to update the references to the above files in the HTML and CSS
- [x] rename class names to use all-lowercase and "dasherized" (a.k.a. kebab-case)

One more significant change I'd like to see regards the structure of the document (I know I mentioned adding a wrapper/container, but I think it's a little bit more involved than that). I'd like to see a containing structure like this:

``` html
<!doctype html>
<html lang="en-us">

<head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">

    <title>Becky Bosshart</title>
    <meta name="description" content="The website of Rebecca Bosshart, author, journalist, teacher, redhead.">

    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <meta name="viewport" content="width=device-width,initial-scale=1">

    <link rel="canonical" href="https://beckybosshart.com/">
    <link rel="stylesheet" type="text/css" href="main.css">
</head>

<body>
    <div class="container">
        <header>
            <!-- main heading and primary nav here -->
        </header>
        <main>
            <section>
                <!-- each section's content here -->
            </section>
            <!-- more sections here -->
        </main>
        <footer>
            <!-- footer and secondary nav goes here -->
        </footer>
    </div>
</body>

</html>
```

One thing to note is that, with this HTML change, the secondary nav will now be at the bottom of the document (where you have "We should put something here") … you're going to have to tweak the CSS a bit (start with `position: fixed;`) to make the footer "stick" to the bottom of the window/screen.

mysterycommand commented on index.html:
```diff
@@ -0,0 +1,116 @@
+    <div class="info">
+        <p>Becky Bosshart is a 2015-2016 Fulbright grantee to Romania and a graduate of the Master's International Creative Writing Program at the University of Nevada, Las Vegas after serving with the Peace Corps in Ukraine from 2012 to 2014. She is the 2016 So To Speak Fiction Contest winner, judged by Emily St. John Mandel, and a 2015 Virginia Quarterly ReviewWriters' Conference fiction schola ...
+    </div>
+    <div class="foot_step"></div>
```
I think we can just use a `hr` tag (stands for "horizontal rule") here?

mysterycommand commented on index.html:
```diff
@@ -0,0 +1,116 @@
+        <p>Becky Bosshart is a 2015-2016 Fulbright grantee to Romania and a graduate of the Master's International Creative Writing Program at the University of Nevada, Las Vegas after serving with the Peace Corps in Ukraine from 2012 to 2014. She is the 2016 So To Speak Fiction Contest winner, judged by Emily St. John Mandel, and a 2015 Virginia Quarterly ReviewWriters' Conference fiction schola ...
+    </div>
+</body>
+<footer>
```
The `footer` goes (somewhat counter intuitively) _inside_ the `body`.

mysterycommand commented:
Once you have a chance to clean up the HTML, I'll probably have a bunch of comments on your CSS. It can feel a little tedious, but this is basically the job! You do some work, someone tells you to do it different, repeat until you're doing it right the first time and telling someone else how to do it.

Of course, we're very close to breaking into working with JS and that's a whole added layer of complexity, but one thing at a time. 😎

Purple906 committed:
Merge branch 'master' into Purple906/mobile-site

Purple906 committed:
updating some of the things outlined in pull request.

updating class names, photo/icon names. location of assets.

mysterycommand commented:
Hey bro! Great start here! One thing I notice is that you now have a directory structure like this:

```
.
├── .babelrc
├── .editorconfig
├── .eslintrc
├── .gitattributes
├── .gitignore
├── .travis.yml
├── LICENSE
├── README.md
├── index.html <──── this shouldn't be here (but it looks like it has your updates)
├── main.css   <──── this shouldn't be here
├── package.json
└── source
    ├── icons
    │   └── facebook.png
    ├── images
    │   ├── bb-photo1.png
    │   ├── bb-photo2.png
    │   ├── bb-photo3.png
    │   └── bb-photo4.png
    ├── index.html <──── … because this is here (but it's missing your updates)
    └── main.css   <──── … because this is here
```

… notice that you have a `./index.html` _and_ a `./source/index.html` (same for `./main.css` and `./source/main.css`). Looks like maybe you just weren't paying too close of attention to which one you were working on because you made updates to `./index.html`, but for the CSS you updated `./source/main.css`.

So I think what you'll want to do is:
- [x] Copy the contents of `./index.html` into `./source/index.html`.
- [x] Delete and remove from Git both `./index.html` and `./main.css` (all your future source files will go into the source directory).
- [x] Update references to images and styles and stuff in `./source/index.html` (because you copied it from `./index.html` and right now everything is relative to that … mostly this should just mean removing "source" from the paths).

Let me know if you get a chance to make these changes and I'll try to look over this again.

Purple906 committed:
get rid of duplicate files

Purple906 committed:
fixed duplicate files and fixed the image and icon tags

Purple906 commented:
okay, I think I've got everything. I added some of the body structure you mentioned. I did have a question about the section ones though, so I didn't add those, but can/will after we talk. let me know what you think.

mysterycommand committed:
style: this is what happens when I run `HTMLPrettify` on the HTML and CSS

mysterycommand committed:
style: always use lower case color values

mysterycommand commented on source/index.html:
```diff
@@ -0,0 +1,119 @@
+
+<head>
+    <!-- <meta charset="utf-8"> -->
+    <!-- <meta http-equiv="x-ua-compatible" content="ie=edge"> -->
```
Uncomment these.

mysterycommand commented on source/index.html:
```diff
@@ -0,0 +1,119 @@
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+
+    <link rel="canonical" href="https://beckybosshart.com/"> -->
```
You can leave these commented for now, but you're going to want to uncomment them eventually. It's gonna mess up your styles at first, but ultimately it's going to give you more control over how the site looks on mobile.

mysterycommand commented on source/index.html:
```diff
@@ -0,0 +1,119 @@
+            <a href="#">Download CV</a>
+            <a href="#">Email</a>
+            <a href="#">Phone</a>
+        </nav>
```
I actually don't think these are `nav`. There really isn't any navigation … maybe [`menu` and `menuitem`](http://webdesign.tutsplus.com/tutorials/introducing-the-html5-menu-and-menuitem-elements--cms-22269) is appropriate here? What do you think?

mysterycommand commented on source/index.html:
```diff
@@ -0,0 +1,119 @@
+            <a class="social-link" href="#"><img src="./icons/facebook.png" /></a>
+        </div>
+        <h2>MENU</h2>
+    </div>
```
Clearly, this should be `menu` and `menuitem`s.

mysterycommand commented on source/index.html:
```diff
@@ -0,0 +1,119 @@
+<footer>
+    <h1>We should put something here</h1>
+    <p>I think stuff in this section, maybe just a link back to the top or a "mirror" of the header</p>
+</footer>
```
This should not be outside the `body`!

mysterycommand commented on source/main.css:
```diff
@@ -0,0 +1,126 @@
+.info {
+    margin: 0px auto;
```
When a CSS value goes to 0 you can just drop the unit (in this case `px`). So just: `margin: 0 auto;`.

mysterycommand commented on source/main.css:
```diff
@@ -0,0 +1,126 @@
+.bb-photo {
+    display: block;
+    width: 100%;
+    margin: 0px auto;
```
No units for 0.

mysterycommand commented on source/main.css:
```diff
@@ -0,0 +1,126 @@
+}
+
+.in-text {
+    margin: 0px;
```
No units for 0.

mysterycommand commented on source/main.css:
```diff
@@ -0,0 +1,126 @@
+.social-home {
+    display: inline-block;
+    width: auto;
+    margin: 0px auto;
```
No units for 0.

mysterycommand commented on source/main.css:
```diff
@@ -0,0 +1,126 @@
+    text-align: center;
+    font-family: Open Sans;
+    font-size: 12px;
+    margin-top: 0px;
```
No units for 0.

mysterycommand commented on source/main.css:
```diff
@@ -0,0 +1,126 @@
+}
+
+a {
+    margin: 0px;
```
No units for 0.

mysterycommand commented on source/main.css:
```diff
@@ -0,0 +1,126 @@
+}
+
+h6 {
+    margin: 0px;
```
No units for 0.

mysterycommand commented on source/main.css:
```diff
@@ -0,0 +1,126 @@
+}
+
+footer {
+    background-color: pink;
```
Avoid color names … I know this is just a place holder, so it's fine for now, but there's a saying in computer-land, "There's nothing more permanent than temporary."

Which just means it's easy to forget about this kind of stuff, and while this is pretty minimal it's a good habit to be pretty vigilant about isolating any "fake" or "temporary" stuff. If you remove the `footer` from the HTML, and forget to remove this style it'll just hang out here waiting to bite you in the ass.

Again, that's a little contrived. This site is super small, but on a bigger project it can be an issue. Maybe just inline-style any temporary HTML?

mysterycommand committed:
style: avoid named colors

mysterycommand committed:
chore: ignore a design folder (where you can put sketch files and other assets)

mysterycommand commented:
Added a bunch more inline comments. The indentation stuff strikes me as weird, but that's how it is in the design so whatever. One other thing that I didn't comment on inline is the `section` stuff. I think you should wrap every `h3` to `.foot-step` in a `section` tag. You could also change the `.foot-step` elements to `hr`s. So (as an example) take this:

``` html
<h3>About</h3>
<div class="info">
    <p>Becky Bosshart blah blah blah …</p>
</div>
<div class="foot-step"></div>
```

… and turn it into this:

``` html
<section>
    <h3>About</h3>
    <div class="info">
        <p>Becky Bosshart blah blah blah …</p>
    </div>
    <hr/>
</section>
```

I think a fair number of the `.info` elements are going to be "events" (or at least time-specific pieces of content) and we should mark them up using microformats (see the [h-event format](http://microformats.org/wiki/h-event)) … just something to think about, not going to work for all of them, but most. While I'm on the subject of microformats, I think this whole site could be made to follow the [h-resume format](http://microformats.org/wiki/h-resume) pretty easily.

Purple906 committed:
uncommented links in header

Purple906 committed:
Merge remote-tracking branch 'origin/Purple906/mobile-site' into Purple906/mobile-site

# Conflicts:
#	source/index.html
#	source/main.css

Purple906 committed:
fixed merge conflicts

Purple906 committed:
Merge remote-tracking branch 'origin/master' into Purple906/mobile-site

mysterycommand committed:
uncommented header stuff, some document structure and CSS fixes

mysterycommand committed:
mysterycommand's changes

mysterycommand committed:
refactor(index): roll purple906's changes over mine

mysterycommand committed:
refactor(main.css): roll Purple906's changes on top of mystercommand's

Purple906 committed:
Merge remote-tracking branch 'origin/master' into Purple906/mobile-site

Purple906 committed:
updates 6/7

added sections, container Div. Also fixed and adjusted some missing
links and CSS tags. Adjusted the margins on the headers to info diffs
so everything is inline and centered. Still need to change “foot-steps”
to HR tags.

Purple906 committed:
updated foot-steps

removed foot-steps and added in HR tags

Purple906 commented:
okay all changes should be made and committed on my end. might want to double check everything since there is a lot in this pull request since we did all that stuff through the terminal last night.

Purple906 committed:
committed testing responsive styles for Nav bar and links

Purple906 committed:
Merge remote-tracking branch 'origin/master' into Purple906/mobile-site

mysterycommand commented on source/main.css:
```diff
@@ -1,9 +1,141 @@
+    font-family: helvetica;
+}
+
+.photo-1 {
```
Let's change this class to `hero-image` … kinda borrowing a convention from Bootstrap.

mysterycommand commented on source/main.css:
```diff
@@ -1,9 +1,141 @@
+    text-align: center;
+}
+
+.bb-photo {
```
Does this just apply to all `img` tags? Maybe just use `img` instead?

mysterycommand commented on source/main.css:
```diff
@@ -1,9 +1,141 @@
+    border-bottom: 2px solid #000000;
+    margin: 5px auto;
+    width: 520px;
+}*/
```
I think you can apply these styles to a `hr` … maybe you don't need the `border-bottom` though?

mysterycommand commented:
Left a couple more comments. Tonight let's get all the content updated, and see if we can get some webfonts working and then merge this to master (I think it's a fine MVP [minimum viable product]). From there we'll create some issues for any fixes. @Thinkinggood, after tonight you should create issues in GitHub for any "bugs" you find or changes you'd like to see.

Purple906 committed:
updated pull request info

Purple906 commented:
done

Purple906 committed:
updated awards section

awards section with new content

Purple906 committed:
removed br tags

removed br tags and added scripts for fonts

Purple906 committed:
updated h6 and h4

updating headers to fit design

Purple906 committed:
updated all of remaining content

updated fiction, non-fiction and teaching experience sections. still
missing content for her time in south korea.

Purple906 committed:
added drop down

added a dropdown menu I found free source online. seems to work fine.
needs some style fixes but CSS and JS copied over well. also still need
to add in the id tags for the sections.

Purple906 committed:
made some adjustments to the hero image

hero image still not right, but I think I’m on the right track.
adjusted the height to auto, removed max-width100% and removed the
margins on the h5 header (read me). image still needs working in large
format, but looks good in mobile

Purple906 committed:
updated from posted issues

removed span tags, adjusted h tag CSS.

mysterycommand committed:
chore: html-prettify index.html

mysterycommand committed:
chore: some whitespace is nice in the head

mysterycommand committed:
style: fix lint errors in CSS, update lint config

mysterycommand committed:
refactor(.info): remove .info divs, apply width/margins to main

mysterycommand committed:
refactor(hero-image/second-nav): move hero-image and second-nave into header

mysterycommand committed:
refactor(dropdown): remove the dropdown, markup, styling, and scripts related to it

mysterycommand committed:
refactor(hero-image): adds the default/smallest image to the markup (in a picture element)

mysterycommand committed:
refactor(hero-image/read-me): re-style the hero-image, picture, and read-me (wrapped in a flex-container now)

mysterycommand committed:
chore: remove old/unused asset

mysterycommand committed:
refactor: remove second-nav css

mysterycommand committed:
Merge branch 'mysterycommand/fix-10' into mysterycommand/fix-11

mysterycommand committed:
refactor: remove h5 styles

mysterycommand committed:
Merge pull request #14 from potpiedigital/mysterycommand/fix-11

Update Hero Image

mysterycommand committed:
Merge pull request #13 from potpiedigital/mysterycommand/fix-10

Remove Dropdown

mysterycommand committed:
Merge pull request #12 from potpiedigital/mysterycommand/fix-8

Remove Unused Markup

Purple906 committed:
updating some copy

small adjustments to Psychopomp
and about me section.

Purple906 committed:
all links are added

currently the flickr links are showing an error (says bad link) when I
tested it, and no flickr link for South Korea. Also no telephone
number. Everything else should be working.

Purple906 committed:
fixed the copy spacing

adjusted copy spacing in the Teaching experience.

Purple906 committed:
adjusted the first sentence

added the link in for the first sentence of the second Paragraph under
the Ukraine teaching experience. Also made it orange.

Purple906 committed:
had some double words

Purple906 committed:
updated sub head

Purple906 committed:
removed the info under south korea

super weird, I definitely did NOT add the paragraph in. but its gone
now.

Purple906 committed:
update CV link and subheads under awards

mysterycommand commented:
Looks great! 👍
